src/output.cpp

```diff
@@ -35,32 +35,19 @@ std::ostream& operator<<(std::ostream& out, MessageType type) {
   return out;
 }
 
-std::string TextOutput::hist_index_label(int power)
+// power must be >= 2<<bits ?
```

Comment:
This is a precondition right? The `?` is confusing - if it's indeed a precondition pls make it an `assert()` in the function

Reply:
added an assert and a lot of explanation (and one example) on how the function works.
Unfortunately the encoding is not terribly intuitive especially because of the initial entry for negative intervals (which is removed here).